<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Windows Live Messenger</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*#MainWindow
{
    	border-image: url(&quot;:/Resources/Message/Background.png&quot;) 0 0 0 0 stretch stretch;
}*/

#SceneBackground
{
    background: rgba(59, 178, 234, 255);
}

#SceneView
{
    border-image: url(&quot;:/Resources/DefaultScene.png&quot;) 0 0 0 0 stretch stretch;
}

#SceneOverlay
{
    background: qlineargradient(x1:0 y1:0 y2:0 y2:1, stop:0 transparent, stop:1 white);
}

#SignInLabel
{
	color: #355a88;
}</string>
  </property>
  <property name="backgroundFileName" stdset="0">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="SignInLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Sign in</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="SceneOverlay">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>128</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="CColorizedNineSlice" name="Background" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="backgroundFileName" stdset="0">
     <string>:/Resources/Message/Background.png</string>
    </property>
    <property name="tintColor" stdset="0">
     <color>
      <red>59</red>
      <green>178</green>
      <blue>234</blue>
     </color>
    </property>
   </widget>
   <widget class="QWidget" name="SceneView" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>401</width>
      <height>128</height>
     </rect>
    </property>
    <property name="tintColor" stdset="0">
     <color>
      <red>59</red>
      <green>178</green>
      <blue>234</blue>
     </color>
    </property>
   </widget>
   <widget class="QWidget" name="SceneBackground" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>128</height>
     </rect>
    </property>
    <property name="tintColor" stdset="0">
     <color>
      <red>59</red>
      <green>178</green>
      <blue>234</blue>
     </color>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CColorizedNineSlice</class>
   <extends>QWidget</extends>
   <header>control/ccolorizednineslice.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
